constructor-manager-ui/src/constructor_manager_ui/main.py

```diff
@@ -28,6 +29,7 @@
     PACKAGES,
     UPDATE_AVAILABLE_VERSION,
 )
+from constructor_manager_ui.style import images  # noqa
```

Comment:
I think there was a comment here explaining the need for this import to load the images resources, maybe could be worthy to readd it too?

Reply:
Whoops, I might have erased by accident. Will fix!